publish_minutes.md

```diff
@@ -14,24 +14,24 @@ Check the following:
 - Ensure aliases have matches (see [people.json file](https://github.com/w3c-ccg/meetings/blob/gh-pages/scribe-tool/people.json))
 
 ## Step 2: Fetch and clean audio (TODO)
-- Fetch the audio file 
-    - For example: `curl -# "https://w3c-ccg.s3.digitalbazaar.com/minutes/2020-08-04-audio.wav" > audio-raw.wav`
-- Open the .wav file in an audio editor (such as audacity) and clean/save the audio:
-    - Delete all audio before the Chair starts talking about the Agenda
-    - Delete all audio after the first person hangs up at the end of the call.
-    - Save the new file as audio.wav or encode to .ogg (file name audio.ogg)
-- Add the cleaned up audio file to github under the meeting date
-    - For example: https://github.com/w3c-ccg/meetings/2020-08-04/audio.ogg
 
-### Step 2b: if you used zoom
+### Step 2a: if you used Jitsi
+- You can use the `download-raw-minutes` tool in the `w3c-ccg/meetings` repository.
+    - For example, to download the audio for 2020-09-29 weekly meeting into `2020-09-29`:
+    `./download-raw-minutes -a 2020-09-29`
+    - For example, to download the audio for 2020-11-02 education meeting into `2020-11-02-vc-education`:
+    `./download-raw-minutes -a -m education 2020-11-02`
 
-Convert the .m4a file to a .ogg file. You'll commit this is step 3 as audio.ogg.
+The `audio.ogg` file will be placed into the correct directory automatically.
 
```

Comment:
```suggestion
Alternatively, instead of running the `download-raw-minutes` script, you can do the steps by hand:

- Create the date directory inside the `meetings` repo, named `yyyy-mm-dd` (eg. `2020-12-01`)
    - if the meeting is for a task force, add a suitable suffix to the directory name, eg `2020-12-01-vc-education`
- Open the audio file in your browser by going to `https://meet.w3c-ccg.org/archives/w3c-ccg-{MEETING NAME}-{DATE}.ogg` and File > Save (ctrl+s or cmd+s). Name the file `audio.ogg`, and save it in the date directory you just created.
    - for a regular CCG meeting, in the URL {MEETING NAME} will be `weekly`. Otherwise, use the task force name, eg. `education`.

```